runtime/runtime.go

```diff
@@ -114,6 +114,25 @@ func Status(failpath string) (string, error) {
 	return t.desc, nil
 }
 
+// StatusCount outputs how many times current term was executed
+func StatusCount(failpath string) (string, error) {
```

Comment:
I would expect that `StatusCount` function should return an `int` instead of `string`. However that can be fixed in future PRs.